iceoryx_posh/test/moduletests/test_roudi_cmd_line_parser.cpp

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
```

Comment:
Please check in all files if the precondition check needs to be `ASSERT_**`. This is the case when you would like to access a value and check beforehand if its available. In this case we require an assertion otherwise we would access an non existing value in the test and the whole unit test could segfault.

Reply:
Btw, it looks like the original code came from PR #523. So these finding was missed in the review of this PR!